components/Wiki/WikiContent.tsx

```diff
@@ -0,0 +1,117 @@
+'use client'
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+import { useState, useEffect } from 'react'
+import { WikiPage } from '../../utils/WikiUtils'
+import Search from '../Search/Search'
+import styles from './WikiContent.module.css'
+
+interface Props {
+    page: WikiPage
+    allPages: WikiPage[]
+}
+
+export default function WikiContent({ page, allPages }: Props) {
+    const pathname = usePathname()
+    const [isSidebarOpen, setIsSidebarOpen] = useState(false)
+    const [isMobile, setIsMobile] = useState(false)
```

Comment:
there is a "useIsMobile" hook in the Hooks.tsx that uses the userAgent to determine if the user is on mobile. Your decision if you want to use the window width instead